src/langchain_google_firestore/document_converter.py

```diff
@@ -60,6 +60,11 @@ def convert_firestore_document(
     if len(page_content) == 1:
```

Comment:
In this case it could be a vector field as well, right?

Can we move the `for` logic removing the vector fields before the conditionals?

```
for k in list(page_content.keys()):
  if isinstance(page_content[k], Vector):
    del page_content[k]

if len(page_content) == 1:
 ...
```

Reply:
Updated